java-tools/OsmAndServer/src/main/java/net/osmand/server/controllers/pub/GpxController.java

```diff
@@ -304,6 +305,69 @@ public ResponseEntity<String> getGpxInfo(@RequestPart(name = "file") @Valid @Not
 		}
 	}
     
+    @PostMapping(path = {"/get-track-data"}, produces = "application/json")
```

Comment:
Post != get-track-data (get wrong word)

Probably should be "process-track-data", "retrieve-?"